sdk/iot/Azure.Iot.Hub.Service/src/Customized/JobRestClient.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using Azure.Core;
+
+namespace Azure.Iot.Hub.Service
+{
+    internal partial class JobRestClient
+    {
+        internal Response<string> CancelImportExportJob(string id, CancellationToken cancellationToken = default)
```

Comment:
From what I remember, the reason we are adding custom logic here is to preserve the raw json text. Could you add a comment for that so we can remember why we need this custom wrapper?